test/CMakeLists.txt

```diff
@@ -29,12 +29,18 @@ else()
 endif()
 
 # ---- Create binary ----
-
-file(GLOB sources CONFIGURE_DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/source/*.cpp)
-add_executable(SmeagleTests ${sources})
+add_executable(SmeagleTests
+	source/main.cpp
+	source/smeagle.cpp
+	source/directionality.cpp
```

Comment:
So if we have hundreds of tests, they would need to be individually listed here? Are you sure we can't do *.cpp (or some other means?)

Reply:
The cmake docs say that file(GLOB) isn't recursive, but it was finding files in subdirectories when I used it. Let's experiment with this.